templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -17,6 +17,10 @@ email address or part of the username of the user you want to @-mention.
 When you begin typing an email address or username, autocomplete suggestions
 will appear.
 
+    !!! tip ""
+        You can use **@all** to mention everyone subscribed to the stream you
+        send the message to.
+
```

Comment:
Since this is possibly something someone will read fairly early in their time on Zulip, we might want to highlight that mentioning someone will give them a notification (how exactly does the notifications for @all mentions work?  I see that they show up in the mentions tab but I'm not sure if they generate missed message emails, etc.)